### ISSUE TITLE
Replace VSCode with VSCodium

### ISSUE THREAD

**Issue by DavideDunne** [equetzal/huronOS-build-tools#24]
During the TCMX2023 if I wanted to open **VSCode** previous to the beginning of the competition and the change of user privileges set to competition, I would get invasive pop-ups from **VSCode** to configure the best options for python and c++ and to install the recommended set of extensions.

That's why I would encourage to replace it for **VSCodium** which is a fork of **VSCode** without the dependencies and telemetry from Microsoft so it might be faster as well.

Basically is chromium or ungoogled chromium but for Visual Studio

https://vscodium.com/
https://youtu.be/rSPpf1zjHG4

**Comment by equetzal** [comment]
I didn't knew about that the distributed version was different than the repository. Be sure that I'll replace vscode with this version and I'll add a symlink to 'code' binary to the 'codium' one :D

**Comment by equetzal** [comment]
Closing with Merge of PR #32

**Comment by DT3264** [comment]
Because of some issues with my git history, the changes disappeared, but now with the merge of PR #44 it is again closed, this time for good jeje
